src/main/java/com/vectorcast/plugins/vectorcastcoverage/VectorCASTPublisher.java

```diff
@@ -266,26 +315,26 @@ private void checkThreshold(Run<?, ?> run,
 						|| isMCDCCoverageOk(action)
 						|| isFunctionCoverageOk(action)
 						|| isFunctionCallCoverageOk(action)){
-					logger.println("[VectorCASTCoverage] [INFO]: Build failed due to coverage percentage threshold exceeds ");
+					logger.println("[VectorCASTCoverage] [FAIL]: Build failed due to a coverage metric fell below the minimum threshold");
 					run.setResult(Result.FAILURE);
 				} 
 				if (isStatementCoverageOk(action)) {
-					logger.println("[VectorCASTCoverage] [INFO]: Statement coverage "+action.getStatementCoverage().getPercentage()+"% < "+healthReports.getMinStatement()+"%.");
+					logger.println("[VectorCASTCoverage] [FAIL]: Statement coverage "+action.getStatementCoverage().getPercentage()+"% < "+healthReports.getMinStatement()+"% threshold.");
 				}
 				if (isBranchCoverageOk(action)) {
-					logger.println("[VectorCASTCoverage] [INFO]: Branch coverage "+action.getBranchCoverage().getPercentage()+"% < "+healthReports.getMinBranch()+"%.");
+					logger.println("[VectorCASTCoverage] [FAIL]: Branch coverage "+action.getBranchCoverage().getPercentage()+"% < "+healthReports.getMinBranch()+"% threshold.");
 				}
 				if (isBasisPathCoverageOk(action)) {
-					logger.println("[VectorCASTCoverage] [INFO]: Basis Path coverage "+action.getBasisPathCoverage().getPercentage()+"% < "+healthReports.getMinBasisPath()+"%.");
+					logger.println("[VectorCASTCoverage] [FAIL]: Basis Path coverage "+action.getBasisPathCoverage().getPercentage()+"% < "+healthReports.getMinBasisPath()+"% threshold.");
 				}
 				if (isMCDCCoverageOk(action)) {
-					logger.println("[VectorCASTCoverage] [INFO]: MC/DC coverage "+action.getMCDCCoverage().getPercentage()+"% < "+healthReports.getMinMCDC()+"%.");
+					logger.println("[VectorCASTCoverage] [FAIL]: MC/DC coverage "+action.getMCDCCoverage().getPercentage()+"% < "+healthReports.getMinMCDC()+"% threshold.");
 				}
 				if (isFunctionCoverageOk(action)) {
-					logger.println("[VectorCASTCoverage] [INFO]: Function Coverage "+action.getFunctionCoverage().getPercentage()+"% < "+healthReports.getMinFunction()+"%.");
+					logger.println("[VectorCASTCoverage] [FAIL]: Function Coverage "+action.getFunctionCoverage().getPercentage()+"% < "+healthReports.getMinFunction()+"% threshold.");
```

Comment:
When writing a test, I noticed the word `Coverage` is capitalized for **function** coverage which is inconsistent with the rest of the coverage type messages.

Maybe extract a function that constructs the message given the coverage type and percentages?

```suggestion
					logger.println("[VectorCASTCoverage] [FAIL]: Function coverage "+action.getFunctionCoverage().getPercentage()+"% < "+healthReports.getMinFunction()+"% threshold.");
```

Reply:
Update made